package.yaml

```
@@ -43,7 +43,7 @@ dependencies:
 - deepseq >= 1.4 && < 1.5
 - filepath
 - http-conduit >= 2.3 && < 2.4
-- hyraxAbif >= 0.2.3.27 && < 0.2.4.0
+- hyraxAbif >= 0.2.4.1
```

а зачем нам жёстко поднимать __нижнюю__ границу? ведь на старом гхц старая версия будет работать. я бы поставил так

```suggestion
- hyraxAbif >=0.2.3.27 && < 0.2.5.0
```